pom.xml

```diff
@@ -57,8 +57,8 @@
         <dep.tempto.version>182</dep.tempto.version>
```

Comment:
Is there any issue that it solves? Can you share more details in the commit message about it?

Do we need to stress test this?

Reply:
https://github.com/docker-java/docker-java/commit/d85ea79eaf7b2a3117f2cf04fac61b354c53e0d2

Reply:
This no longer affects us, right?

Anyway, good to add something to commit message (worth mentioning whether this currently affects us or not).

> Do we need to stress test this?

I think this may be a good idea, as we're pulling in more than one code change (are we?)

Reply:
I'll run that multiple times then